big_tests/tests/gdpr_SUITE.erl

```diff
@@ -127,19 +153,39 @@ init_per_group(remove_personal_data_with_mods_disabled, Config) ->
 init_per_group(retrieve_personal_data_pubsub, Config) ->
     dynamic_modules:ensure_modules(domain(), pubsub_required_modules()),
     Config;
+init_per_group(retrieve_personal_data_inbox = GN, Config) ->
+    init_inbox(GN, Config, inbox_opts());
 init_per_group(_GN, Config) ->
     Config.
 
 end_per_group(_GN, Config) ->
     Config.
 
 init_per_testcase(retrieve_inbox = CN, Config) ->
-    init_inbox(CN, Config);
+    init_inbox(CN, Config, inbox_opts());
 init_per_testcase(remove_offline = CN, Config) ->
     offline_started(),
     escalus:init_per_testcase(CN, Config);
-init_per_testcase(retrieve_inbox_for_multiple_messages = CN, Config) ->
-    init_inbox(CN, Config);
+init_per_testcase(remove_inbox = CN, Config) ->
+    init_inbox(CN, Config, inbox_opts());
+init_per_testcase(CN, Config) when
+      CN =:= remove_inbox_muclight ;
+      CN =:= retrieve_inbox_muclight ->
+    Host = ct:get_config({hosts, mim, domain}),
+    dynamic_modules:ensure_modules(Host, [{mod_muc_light,
+                                           [{host, binary_to_list(?MUCHOST)},
+                                            {backend, mongoose_helper:mnesia_or_rdbms_backend()},
+                                            {rooms_in_rosters, true}]}]),
+    init_inbox(CN, Config, inbox_opts());
+init_per_testcase(CN, Config) when CN =:= retrieve_inbox_muc;
+                                   CN =:= remove_inbox_muc ->
+    muc_helper:load_muc(muc_domain()),
+    Config1 = escalus:create_users(Config, escalus:get_users([alice, bob])),
+    Users = ?config(escalus_users, Config1),
+    Alice = lists:keyfind(alice, 1, Users),
+    RoomName = muc_helper:fresh_room_name(),
+    Config2 = muc_helper:start_room(Config1, Alice, RoomName, <<"some_friendly_name">>, default),
```

Comment:
those lines could be move to separate function, e.g `given_room_started` and It could be actually started in test cases instead of init_per_tesetscases`.